components/event-sources/reconciler/object/equality.go

```diff
@@ -78,8 +78,8 @@ func channelEqual(c1, c2 *messagingv1alpha1.Channel) bool {
 	return true
 }
 
-// ksvcEqual asserts the equality of two Knative Service objects.
-func ksvcEqual(s1, s2 *servingv1alpha1.Service) bool {
+// deploymentEqual asserts the equality of two Deployment objects.
+func deploymentEqual(s1, s2 *appsv1.Deployment) bool {
```

Comment:
just wondering, isn't there a deep equality check function in k8s client-go lib?

Reply:
I am happy to remove the self-written equality check if there is one in k8s. I did a quick search and didn't find one. If you know where it is, please let me know :)

Reply:
I'm not so sure, have you used this package? https://godoc.org/k8s.io/apimachinery/pkg/api/equality

Reply:
I will give it a try next time and would like to not introduce another change :)